data-source/jsonnet/northern-ireland/census_household.jsonnet

```diff
@@ -1,31 +1,101 @@
 // Accommodation
-local accommodation_introduction = import 'blocks/household/accommodation/accommodation_introduction.jsonnet';
-local accommodation_summary = import 'blocks/household/accommodation/accommodation_summary.jsonnet';
-local accommodation_type = import 'blocks/household/accommodation/accommodation_type.jsonnet';
-local adapted = import 'blocks/household/accommodation/adapted.jsonnet';
-local central_heating = import 'blocks/household/accommodation/central_heating.jsonnet';
-local number_bedrooms = import 'blocks/household/accommodation/number_bedrooms.jsonnet';
-local number_of_vehicles = import 'blocks/household/accommodation/number_of_vehicles.jsonnet';
-local own_or_rent = import 'blocks/household/accommodation/own_or_rent.jsonnet';
-local renewables = import 'blocks/household/accommodation/renewables.jsonnet';
-local self_contained = import 'blocks/household/accommodation/self_contained.jsonnet';
-local type_of_flat = import 'blocks/household/accommodation/type_of_flat.jsonnet';
-local type_of_house = import 'blocks/household/accommodation/type_of_house.jsonnet';
-local who_rent_from = import 'blocks/household/accommodation/who_rent_from.jsonnet';
+local accommodation_introduction = import 'household/blocks/accommodation/accommodation_introduction.jsonnet';
+local accommodation_summary = import 'household/blocks/accommodation/accommodation_summary.jsonnet';
+local accommodation_type = import 'household/blocks/accommodation/accommodation_type.jsonnet';
+local adapted = import 'household/blocks/accommodation/adapted.jsonnet';
+local central_heating = import 'household/blocks/accommodation/central_heating.jsonnet';
+local number_bedrooms = import 'household/blocks/accommodation/number_bedrooms.jsonnet';
```

Comment:
This file doesn't exist.

Reply:
Ah, these were already existing, removed now. If only we had a linter 👀